dataproc/list_clusters.py

```diff
@@ -10,49 +10,53 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Sample command-line program to list Cloud Dataproc clusters in a region.
 
-""" Sample command-line program for listing Google Dataproc Clusters
-"""
+Example usage:
+python list_clusters.py --project_id=my-project-id --region=global
 
+"""
 import argparse
-
-import googleapiclient.discovery
+from google.cloud import dataproc_v1
```

Comment:
Nit: We group imports for built-ins separately from imports for pip packages.

Reply:
Done.

Reply:
Did you forget to push your branch? I don't see any commits from today.

Reply:
I committed latest files on 5-17. Please let me know if you don't see the latest changes.